testing/test_capture.py

```diff
@@ -470,6 +470,38 @@ def test_hello(capfdbinary):
         """)
         reprec.assertoutcome(passed=1)
 
+    @pytest.mark.skipif(
+        sys.version_info < (3,),
+        reason='only have capsysbinary in python 3',
+    )
+    def test_capsysbinary(self, testdir):
+        reprec = testdir.inline_runsource("""
+            def test_hello(capsysbinary):
+                import sys
+                # some likely un-decodable bytes
+                sys.stdout.buffer.write(b'\\xfe\\x98\\x20')
```

Comment:
> sys.stdout.buffer.write(b'\\xfe\\x98\\x20')

Is it intentional having to write bytes here?
(https://github.com/pytest-dev/pytest/issues/6871)